chain-signatures/node/src/protocol/signature.rs

```diff
@@ -356,13 +357,13 @@ impl SignatureManager {
     #[allow(clippy::result_large_err)]
     fn retry_failed_generation(
         &mut self,
-        receipt_id: ReceiptId,
+        sign_request_identifier: SignRequestIdentifier,
```

Comment:
this is quite long now as a variable name. Let's just simplify all of them to just `request_id: SignRequestIdentifier`

Reply:
I'll do all the rafactoring in an upcoming PR. Want to prioritize making batch multiple sign possible on dev first.